--- /dev/null
+++ b/openwrt/target/linux/mediatek/mt7622/base-files/etc/config/network
@@ -0,0 +1,34 @@
+config interface 'loopback'
+	option device 'lo'
+	option proto 'static'
+	option ipaddr '127.0.0.1'
+	option netmask '255.0.0.0'
+
+config device
+	option name 'br-lan'
+	option type 'bridge'
+	list ports 'eth0'
+	list ports 'lan1'
+	list ports 'lan2'
+	list ports 'lan3'
+	list ports 'lan4'
+
+config bridge-vlan
+	option device 'br-lan'
+	option vlan '31'
+	list ports 'eth0:t'
+	list ports 'lan1:u*'
+
+config bridge-vlan
+	option device 'br-lan'
+	option vlan '100'
+	list ports 'lan2:u*'
+	list ports 'lan3:u*'
+	list ports 'lan4:u*'
+
+config interface 'MGMT'
+	option proto 'static'
+	option device 'br-lan.31'
+	option ipaddr '192.168.31.1'
+	option netmask '255.255.255.0'
+	option broadcast '192.168.31.255'
